README.md

```diff
@@ -184,68 +204,82 @@ parameter a `Term`. Their ASTs are defined
 ### Printing
 
 The [`printer`](/src/main/scala/smtlib/printer) helps with printing out SMT-LIB
-complient commands. This means that the output of a printer can be send
+compliant commands. This means that the output of a printer can be sent
 directly to an SMT solver.
 
 ### Standard Theories
 
-Finally the [`theories`](/src/main/scala/smtlib/theories) module provides tree
-builders to create theory-specific formulas. Each theory module provides
-`apply` and `unapply` methods on various object to manipulate the `Term`
-representing the actual theory expression.
+Finally, the [`theories`](/src/main/scala/smtlib/theories) module provides tree
+builders to create theory-specific formulas. Each theory module provides `apply`
+and `unapply` methods on various object to manipulate the `Term` representing
+the actual theory expression.
 
 Development
 -----------
 
-The library is still under development and the API will likely go through quite a few
-changes. It was originally part of [CafeSat](https://github.com/regb/scabolic)
-and has been made standalone in order for the
-[Leon](https://github.com/epfl-lara/leon) project to rely on it.
+The library is still under development and the API will likely go through quite
```

Comment:
Is it haha?

Reply:
:see_no_evil:

Reply:
We will update it to smtlib 3 someday :copium: